cac-prod.yml

```diff
@@ -23,7 +23,7 @@ jenkins:
         name: all
     - buildMonitor:
         includeRegex: >-
-          ^HMCTS_.*(Platform|CNP).*\/(draft-store|cmc-pdf-service|feature-toggle-api|private-beta-invitation-service|service-auth-provider-app|spring-boot-template|cnp-rhubarb-recipes-service)\/master
+          ^HMCTS_.*(Platform|CNP).*\/(cmc-citizen-frontend|draft-store|cmc-pdf-service|feature-toggle-api|private-beta-invitation-service|service-auth-provider-app|spring-boot-template|cnp-rhubarb-recipes-service)\/master
```

Comment:
this is the platform team build monitor, you want your own one

Reply:
oops - blindly adding